packages/ast-tooling/index.ts

```diff
@@ -16,7 +16,7 @@ import {
 import * as fleece from 'silver-fleece';
 import * as Walker from 'zimmerframe';
 import type { namedTypes as AstTypes } from 'ast-types';
-import type * as AstKinds from 'ast-types/gen/kinds.d.ts';
+import * as AstKinds from 'ast-types/gen/kinds';
```

Comment:
I had to update the import path here after updating the package, I'm not sure if this breaks anything but it seemed fine to me